### PR TITLE
Rename team messages to scratchpad

### PR DESCRIPTION
**Pull request by adbharadwaj**
The name “Team Messages” misleads the experts to start communicating through Team messages google document. Instead, the best way to communicate is To-dos and Slack (if required) as much as possible. As a solution, we want to rename the document to Scratchpad.

### CODE REVIEW

File: orchestra/static/orchestra/common/components/team-messages/team-messages.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<website-iframe title="Team Messages"
+<website-iframe title="Scratchpad"
 		id="team-messages"
```
Comment (marcua) [inline orchestra/static/orchestra/common/components/team-messages/team-messages.html]: if you care :)

File: docs/source/api.rst [hunk 2]
```diff
@@ -54,7 +54,7 @@ All requests must be signed using `HTTP signatures <http://tools.ietf.org/html/d
               "id": 123,
```
Comment (marcua) [inline docs/source/api.rst:54]: if you `git grep -i team|grep messages` in the root of this repository, does anything relevant pop up?

Comment (adbharadwaj) [reply docs/source/api.rst:54]: Screen Shot 2021-04-19 at 4 45 58 PM

Comment (adbharadwaj) [reply docs/source/api.rst:54]: aah that was so much better grep command than what i was running :P

File: orchestra/static/orchestra/common/components/team-messages/team-messages.html (hunk 1 above)
Comment (adbharadwaj) [reply orchestra/static/orchestra/common/components/team-messages/team-messages.html]: care++!!

File: docs/source/api.rst (hunk 2 above)
Comment (marcua) [reply docs/source/api.rst:54]: ooo perfect! and i just caught a small bug in my grep: `git grep -i team|grep -i messages` (added a second -i). hopefully it doesn't uncover anything!

File: orchestra/migrations/0094_auto_20210419_1747.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.13 on 2021-04-19 17:47
```
Comment (paopow) [inline orchestra/migrations/0094_auto_20210419_1747.py]: rename the file to `0094_rename_team_messages_url.py` or sth along the same line?

File: docs/source/api.rst (hunk 2 above)
Comment (adbharadwaj) [reply docs/source/api.rst:54]: The new grep was showing changes from these files:
```
docs/source/bots.rst
orchestra/frontend/build/static/js/2.564c31ea.chunk.js.map
```

I checked and found nothing relevant.

**Review by paopow — APPROVED** [review]

**Pull request merged**